lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2283,8 +2297,17 @@ private static function get_block_nodes( $theme_json, $selectors = array() ) {
 				'selectors'  => $feature_selectors,
 				'duotone'    => $duotone_selector,
 				'variations' => $variation_selectors,
+				// 'state_selectors' => $selectors[ $name ]['.current-item'],
```

Comment:
It may be that we need to create another set of sub-selectors under a new key. This would house any further "state" selectors like `current-item`.